docker/zarr-output/create_zarr_optimus.py

```diff
@@ -463,14 +464,15 @@ def create_zarr_files(args):
     Args:
         args (argparse.Namespace): input arguments for the run
     """
-    version = "1.0.0"
+    version = "1.0.1"
 
     # initiate the zarr file
     root_group = init_zarr(
```

Comment:
I think it would be good practice to convert all these to named arguments as opposed to positional